### DISCUSSION TITLE
DatetimePicker组件支持默认选中当前时间

### DISCUSSION THREAD

**Discussion by seame1on** [3lang3/react-vant#617] [General]
DatetimePicker组件在初始状态未设置value时，弹层不会自动选中当前时间，
对比了下 antd-mobile 的 DatePicker，未设置值时弹出层默认自动选中当前时间
期望能在未选择时间时，弹出层能选中当前时间，感谢！
见 [demo](https://codesandbox.io/s/datepicker-0vvtqi?file=/App.tsx) 对比

**Comment by 3lang3** [comment]
合理，下次发版前改下好了，也欢迎👏🏻pr哈
